README.md

```diff
@@ -1,38 +1,37 @@
 # JavaScript Basics in Express
 
-## Getting started
+This repository contains a series of controller functions I wrote using Express to create API endpoints for an application. It uses functions from my JavaScript Basics code, which are called when each controller function runs.
```

Comment:
Nice readme! Id suggest you add a link to your linkedIn profile somewhere, Nice readme!
If you want to improve the look of it a bit more, also it would be awesome if you could add some details about the routes and what they do here, a nice way to do this would be in a table, something like:
```
### Strings

| Route | HTTP Verb | Description |
| ----------- | ----------- | ----------- |
| `/strings/hello/:string` | GET | Returns `Hello ${string}` | 
| `/strings/upper/:string` | GET | Returns the string in all uppercase |
```
Theres more info on extended formatting rules fro READMEs available [here](https://www.markdownguide.org/cheat-sheet/#extended-syntax)